PowerPointLabs/PowerPointLabs/EffectsLab/View/EffectsLabBlurDialogBox.xaml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Windows;
+
+using PowerPointLabs.ActionFramework.Common.Log;
+
+namespace PowerPointLabs.EffectsLab.Views
+{
+    /// <summary>
+    /// Interaction logic for EffectsLabBlurDialogBox.xaml
+    /// </summary>
+    public partial class EffectsLabBlurDialogBox
+    {
+        public delegate void UpdateSettingsDelegate(int percentage, bool isTinted);
+        public UpdateSettingsDelegate SettingsHandler { get; set; }
+
+        private string currentFeature;
+        private float lastBlurriness;
+
+        public EffectsLabBlurDialogBox()
+        {
+            InitializeComponent();
+        }
+        
+        public EffectsLabBlurDialogBox(string feature)
+            : this()
+        {
+            currentFeature = feature;
+            string properFeatureName = "Effects Lab";
+
+            switch (feature)
+            {
+                case TextCollection.EffectsLabBlurrinessFeatureSelected:
+                    properFeatureName = TextCollection.EffectsLabBlurSelectedButtonLabel;
+                    lastBlurriness = EffectsLabBlurSelected.CustomPercentageSelected;
+                    tintCheckbox.IsChecked = EffectsLabBlurSelected.IsTintSelected;
+                    break;
+                case TextCollection.EffectsLabBlurrinessFeatureRemainder:
+                    properFeatureName = TextCollection.EffectsLabBlurRemainderButtonLabel;
+                    lastBlurriness = EffectsLabBlurSelected.CustomPercentageRemainder;
+                    tintCheckbox.IsChecked = EffectsLabBlurSelected.IsTintRemainder;
+                    break;
+                case TextCollection.EffectsLabBlurrinessFeatureBackground:
+                    properFeatureName = TextCollection.EffectsLabBlurBackgroundButtonLabel;
+                    lastBlurriness = EffectsLabBlurSelected.CustomPercentageBackground;
+                    tintCheckbox.IsChecked = EffectsLabBlurSelected.IsTintBackground;
+                    break;
+                default:
+                    Logger.Log(feature + " does not exist!", ActionFramework.Common.Logger.LogType.Error);
+                    break;
+            }
+
+            Title = properFeatureName + " Settings";
+
+            tintCheckbox.Content = properFeatureName.Replace("Blur", "Tint");
```

Comment:
Instead of using replace, why not create new const in TextCollection? It does not make much sense to replace blur with tint